---
title: Genomic analysis of COVID-19 spread in Haiti.
authors:
  - Jeffrey Tran-


license: "CC-BY"  
licenseLink: "https://creativecommons.org/licenses/by/4.0/"
dataset: "https://nextstrain.org/community/poisedleft/ncov/haiti/build?c=emerging_lineages&d=map&p=full"

abstract: "Haiti is currently experiencing its worse outbreak of the pandemic, driven primarily by the P.1 variant. This is an early look at sequences released by the country's National Public Health Laboratory (Laboratoire National de Sante Publique) to GISAID. After sporadic sequencing through 2020 and early 2021 the lab released about 40 sequences taken in May, at about the time the epidemic was noticebly picking up." 
---

This particular analysis uses sequences focused in South America (most relevant for transmission) as well as those genetically close to ones found in Haiti. The number of sequences can be seen in the map at right. Keep in mind that this can influence visualizations and/or inferred trees. The dominant trend in this region of the world has been the emergence of P.1 as the dominant lineage. 

# [sample dates](http://nextstrain.org/community/poisedleft/ncov/haiti/build?d=tree&c=num_date&f_country=Haiti&l=radial)

Sampling has been somewhat limited. However from what we can see the most recent sequences are from disjoint lineages than earlier samples. This is consistant with what has happened all over the world, with Variants of Concern replacing previous lineages.. 

# [emerging_lineages](http://nextstrain.org/community/poisedleft/ncov/haiti/build?d=tree&c=emerging_lineage&f_country=Haiti&l=radial)

.. as we can see here. One of these samples is B.1.1.7, though it can't be ruled out as more prevalant without broader sequencing and specific analysis of sampling strategy. One is wild type. The rest are P.1. 

# [P.1](http://nextstrain.org/community/poisedleft/ncov/haiti/build?c=country&d=tree&clade=20J/501Y.V3&f_country=Haiti,Brazil,Argentina,Uraguay&f_clade_membership=20J/501Y.V3&l=radial)

There are a handful of introductions of P.1. Unsurprisingly, many are genetically similar to branches originating in Brazil/South America. That was the most likely possibility but is good to confirm. Additional analysis could consist of analyzing volume of air travel combined with presence of variants, both backwards looking to supplement data of this type, and forward looking, to evaluate future risks.  

Many introductions have only a single sequence, so not much can be inferred. Many sequences however come from one, in country clade.. 


todo: make better colors

todo: get transmission lines to work in the map. Is this a build issue? 


# [In Country Transmission](https://nextstrain.org/community/poisedleft/ncov/haiti/build?branchLabel=emerging_lineage&d=tree&ci&f_haiti=yes&clade=20J/501Y.V3&gt=nuc.2455T&tl=country&l=radial)

This batch of samples seems to have mostly come from a single introduction, estimated to have occured between Feb 4, 2021 and April 6, 2021, with the variant spreading through local transmission up to the present day (the grey horizontal bars represent the model's estimated range of an inferred common ancestor). 

 
As there are no actual samples from those dates, this represents modelling assumptions about the rate at which mutations occur and the most likely sets of common ancestors. In this case there are a fair number of samples in the clade with varying levels of genetic diversity so the general pictures should be sound, especially when combined with real world travel patterns and epidemiological conditions. Covid cases led by P.1 were surging in Brazil at the beginning of the year, and there are direct flights from Sao Paulo to Port-au-Prince.

(presentation note.. I might have preferred to be zoomed out a node or two. However as these had no defining mutations I was not sure of the best way to do this.. explicitly including the mutations worked, but was more of a workaround here). 


# [Implications](https://nextstrain.org/community/poisedleft/ncov/haiti/build?c=emerging_lineages&d=map&p=full)

* Genetic sequencing can provide valuable insight into the nature and trajectory of an epidimic however resources have been typically limited. It is essential to support local laboratories in their efforts.  

* The timing and severity of an outbreak are set in motion months before it becomes evident. While this is an incomplete picture, it is consistent with what we might expect from general epidemiological dynamics.

* It's possible/probable that had genetic sequencing been available earlier the presence and spread of the variant may have alerted public health authorities to the threat sooner allowing them to take proper measures such as activating public health resources, implementing restrictions, etcetera. 

* In the case of Haiti while P.1 is likely to continue to be devastating, B.1.617.2 (Delta) looms as yet another threat. One way to monitor this might be to use more detailed flight data along with local prevalance of the variant in those regions as a dashboard to inform importation risk assessments. 





